python/nav/etc/sortedstats.conf

```diff
@@ -1,20 +1,7 @@
 # Configfile for sortedStats webutility
 
-# Make your own views if you want. Make sure:
-# - that the viewname (the one in brackets) and the name is not taken
-# - that name, path and dsdescr are set.
-#
-
-# NB: You may experience problems when editing this file and then reloading
-# the ranked statistics-page. This is because the threads in the webserver
-# still uses the old file and not the new one. Waiting or reloading the page
-# several times will fix the problem.
-
-# The values here are in seconds. If someone wants to see a view and
-# the cache is older than this, it will fetch live data.
-
-# Define reports to create periodically for sortedstats
-# different timeframes are updated at different intervals
+# Define reports to create periodically for sortedstats.
+# Different timeframes are updated at different intervals:
```

Comment:
I repurposed the file to configure what should be periodically cached, so removing it would cause issues. Are you saying it should be renamed something else?

Reply:
I think that's a good idea, and so you did :)